### DISCUSSION TITLE
How to setup the Google analytics ID?

### DISCUSSION THREAD

**Discussion by diegoalarc** [MrGreensWorkshop/MrGreen-JekyllTheme#101] [How to] [answered]
Could you explain bette how to setup the Google analytics ID?

**Comment by MrGreensWorkshop** [comment]
Just put your Google analytics ID to `google_analytics_id: ""` for each language in [_data/owner/](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/_data/owner/) folder.

I think what you suggest is how to create an Google analytics ID etc. That's long shot. Can't explain it by writing it. Google is your friend. I think that needs a video.

**Reply by diegoalarc** [reply]
>Just put your Google analytics ID to google_analytics_id: "" for each language in [_data/owner/](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/_data/owner/) folder.

I did that many times......Could you give me a hint? I am sure the code I am using is activate and works but the web is not sending anything. Thanks for the answer.

**Comment by MrGreensWorkshop** [comment, marked as answer]
It's have been long time since I set up those. One of these could help.

Please check [main.yml](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/_data/conf/main.yml), search this and add your domain. 
```
      # same as domain in _config.yml file
      cookie_domain: ""
```

When you open the web page, you should open consent window and switch on `Performance cookies` that enables `analytics_storage`.

You can try to debug your website using this.
https://support.google.com/tagassistant/answer/10039345?hl=en

I think that's all I can tell.
